torch/distributed/__init__.py

```diff
@@ -4,6 +4,12 @@
 
 
 def is_available():
+    """
+    Returns ``True`` if the distributed package is available. Otherwise,
+    ``torch.distributed`` does not expose any other APIs. Currently,
+    ``torch.distributed`` is available on Linux and MacOS. Set
+    ``USE_DISTRIBUTED=1`` at compile time to enable it.
```

Comment:
It might be helpful to specify that this is for building from source?
Also, what's the current default?

Reply:
good point! will update

current default: USE_DISTRIBUTED=1 on Linux and USE_DISTRIBUTED=0 on MacOS, let me include those